Dockerfile.proxy

```diff
@@ -0,0 +1,2 @@
+FROM registry.redhat.io/rhel7:latest
+LABEL io.openshift.release.operator true
```

Comment:
We should add a file into the image to say what this is being used for. This will also make the image hash unique.

Reply:
Add a readme file under docs/, does this make sense?